modules/RTC/TraceablePeerConnection.js

```diff
@@ -1424,7 +1424,7 @@ const normalizePlanB = function(desc) {
  * @returns {Array<Object>} ssrcLines with removed lines referencing msid "-".
  */
 function replaceDefaultUnifiedPlanMsid(ssrcLines = []) {
-    if (!browser.isChrome() || !browser.isVersionGreaterThan(70)) {
+    if (!browser.isChromiumBased() || !browser.isEngineVersionGreaterThan(70)) {
```

Comment:
Can you pl drop the second part of the check since our mimum supported version is 72?